src/custom/constants/index.ts

```diff
@@ -62,6 +62,8 @@ export const DEFAULT_ORDER_DELAY = 20000 // 20s
 export const PENDING_ORDERS_BUFFER = 60 * 1000 // 60s
 export const CANCELLED_ORDERS_PENDING_TIME = 5 * 60 * 1000 // 5min
 export const PRICE_API_TIMEOUT_MS = 10000 // 10s
+export const EXPIRED_ORDERS_BUFFER = 45 * 1000 // 45s
+export const CHECK_EXPIRED_ORDERS_INTERVAL = 10000 // 10 sec
```

Comment:
I think I've removed these when doing the cancellation.
Better check they are used anywhere, because I think they are not.

Reply:
https://github.com/gnosis/cowswap/pull/842